snuba/datasets/storages/metrics.py

```diff
@@ -143,6 +143,23 @@ def ignore_policy_closure() -> DeadLetterQueuePolicy:
     write_format=WriteFormat.VALUES,
 )
 
+org_counters_storage = ReadableTableStorage(
+    storage_key=StorageKey.METRICS_COUNTERS,
+    storage_set_key=StorageSetKey.METRICS,
+    schema=TableSchema(
+        local_table_name="metrics_counters_v2_local",
+        dist_table_name="metrics_counters_v2_dist",
+        storage_set_key=StorageSetKey.METRICS,
+        columns=ColumnSet(
+            [
+                *aggregated_columns,
+                Column("value", AggregateFunction("sum", [Float(64)])),
+            ]
+        ),
+    ),
+    query_processors=[ArrayJoinKeyValueOptimizer("tags"), TableRateLimit()],
```

Comment:
Not sure if I need the `ArrayJoinKeyValueOptimizer` here, I assume it doesn't hurt.

Reply:
I think it's probably safer to leave it, in case somebody wants to use tags later